src/web/handlers/GestureHandler.ts

```diff
@@ -38,7 +40,7 @@ export default abstract class GestureHandler {
   protected awaiting = false;
   protected active = false;
   protected shouldResetProgress = false;
-  protected pointerType: PointerType = PointerType.NONE;
+  protected pointerType: WebPointerType = WebPointerType.NONE;
```

Comment:
Wouldn't it make more sense to just use `PointerType` and remove `WebPointerType` entirely at this point? You could just map it when building an intermediate event instead of having two separate enums doing exactly the same thing.

Reply:
Done in [407bbbe](https://github.com/software-mansion/react-native-gesture-handler/pull/2760/commits/407bbbe216a9c60db9fa1460c5e3216dd9ebfd4b)